Packs/Twitter/Integrations/Twitterv2/Twitterv2.py

```diff
@@ -0,0 +1,359 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+import urllib3
+from typing import Dict, Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def tweet_search(self, query: str, start_time: Optional[str],
+                     end_time: Optional[str], limit: Optional[int],
+                     next_token: Optional[str]) -> tuple[dict, List[dict], Optional[str]]:
+        """ Gets tweets according to the query.
+            Args:
+                query: str - The query from the user.
+                start_time: str - Start date from which the Tweets will be provided.
+                end_time: str - The most recent date to which the Tweets will be provided.
+                limit: int - Maximum number of results to return.
+                next_token: str - A value that encodes the next 'page' of results that can be requested.
+
+            Returns:
+                A tuple[dict, dict, Optional[str]] with:
+                List[dict]: raw response.
+                List[dict]: context data.
+                str: next token.
+        """
+        result: List[dict] = []
+        query_params = {'query': ''.join(f'"{item}"' for item in query),
+                        'tweet.fields': 'id,text,attachments,author_id,conversation_id,created_at,public_metrics',
+                        'expansions': 'attachments.media_keys,author_id',
+                        'media.fields': 'media_key,type,url,public_metrics,alt_text',
+                        'user.fields': 'id,name,username,created_at,description,verified',
+                        'max_results': f'{limit}',
+                        'start_time': start_time,
+                        'end_time': end_time,
+                        'next_token': next_token}
+        response = self._http_request(method="GET", url_suffix='/tweets/search/recent',
+                                      headers=self._headers, params=query_params,
+                                      ok_codes=[200])
+        result, next_token = create_context_data_search_tweets(response)
```

Comment:
This should be in the command itself. the create_context_data....
The request should return the results as is